src/Cryptol/Eval/SBV.hs

```diff
@@ -302,7 +303,7 @@ instance Backend SBV where
        do unless (0 <= e) (raiseError sym NegativeExponent)
           pure $! SBV.svExp a b
     | otherwise =
-       raiseError sym (UnsupportedSymbolicOp "integer exponentation")
+       liftIO (X.throw (UnsupportedSymbolicOp "integer exponentiation"))
```

Comment:
Yes. I'm very happy with this particular change. While I was investigating #725, I was thinking that `raiseError` was the wrong thing to use in this situation. We should only be using `raiseError` for functions that are actually partial according to our denotational semantics, and "symbolic" vs "concrete" is not a concept that exists in our denotational semantics.